planet/clients/data.py

```diff
@@ -175,11 +175,12 @@ async def create_search(self,
         url = self._searches_url()
 
         # TODO: validate item_types
-        request_json = {
+        request_json: typing.Dict[str, typing.Any] = {
             'name': name, 'filter': search_filter, 'item_types': item_types
         }
+        # QUESTION: can we just set the value to True or False above?
         if enable_email:
-            request_json['__daily_email_enabled'] = True
+            request_json['__daily_email_enabled'] = enable_email
```

Comment:
I'm curious about this change. What does it improve?

Reply:
I'm addressing both of these questions by setting `__daily_email_enabled: enable_email` in the literal dict expression at line 179.